zip/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: install zip and unzip
+  ansible.builtin.apt:
+    pkg:
+      - zip
+      - unzip
+    state: present
```

Comment:
I'd add this cache stuff and error checking https://github.com/Chia-Network/ansible-roles/blob/main/git/tasks/main.yml#L7:L11

Also probably want `become: yes` (or true, or whatever)

Reply:
doing `become: true` just to make the linters happy.  Pushed commits to address this.